scripts/manage_study.py

```diff
@@ -140,370 +146,47 @@ def download_from_bucket(file_path):
     return destination
 
 def validate_metadata_file(metadata_path):
-    placeholder = 'SCP555' # TODO: Refactor CellMetadata to not require this
-    metadata = CellMetadata(metadata_path, '', '', study_accession=placeholder)
-    print(f'Validating {metadata_path}')
-
-    convention_path = download_from_bucket(IngestPipeline.JSON_CONVENTION)
-    with open(convention_path) as file:
-        convention = json.load(file)
-
-    validate_input_metadata(metadata, convention)
-    serialize_issues(metadata)
-    report_issues(metadata)
-    exit_if_errors(metadata)
+    study_name = get_parsed_args().study_name
+    dry_run = get_parsed_args().dry_run
+    verbose = get_parsed_args().verbose
+    study_accession_res =get_connection().get_study_attribute(
+        study_name=study_name,
+        attribute='accession',
+        dry_run=dry_run)
+    study_file = ObjectId('addedfeed000000000000000')
```

Comment:
are these just mock ids, or do they need to correspond to an actual study in the environment?  add a comment either way explaining

Reply:
9fe8324